scripts/tests

```diff
@@ -46,10 +49,14 @@ def base_run(bin, force_qemu, verbose):
     else:
         logging.debug(f"Running {bin} natively")
 
+        args = ["make", f"{bin}"]
+        logging.info(' '.join(args))
+
         p = subprocess.run(
-            ["make", f"{bin}"],
+            args,
             stdout=subprocess.DEVNULL if not verbose else None,
```

Comment:
Is `verbose` set in the CI?

Reply:
yes, it is set in this commit https://github.com/pq-code-package/mlkem-c-aarch64/pull/60/commits/8048dd7d917cb82ec6a14b5dc6f84192710c66bf

Reply:
Hm ok -- why is it then that the CC command line is missing from 
```
INFO  > make CROSS_PREFIX=aarch64-none-linux-gnu- test/bin/gen_KAT1024
  CC      
mlkem/poly.c: In function ‘pqcrystals_kyber1024_ref_scalar_compress_q_16’:
mlkem/poly.c:27: warning: ignoring ‘#pragma CPROVER check’ [-Wunknown-pragmas]
   27 | #pragma CPROVER check push
      | 
mlkem/poly.c:28: warning: ignoring ‘#pragma CPROVER check’ [-Wunknown-pragmas]
   28 | #pragma CPROVER check disable "unsigned-overflow"
      | 
mlkem/poly.c:30: warning: ignoring ‘#pragma CPROVER check’ [-Wunknown-pragmas]
   30 | #pragma CPROVER check pop
      | 
mlkem/poly.c: In function ‘pqcrystals_kyber1024_ref_scalar_compress_q_32’:
mlkem/poly.c:[64](https://github.com/pq-code-package/mlkem-c-aarch64/actions/runs/9543519253/job/26300276213?pr=60#step:5:65): warning: ignoring ‘#pragma CPROVER check’ [-Wunknown-pragmas]
   64 | #pragma CPROVER check push
      | 
mlkem/poly.c:[65](https://github.com/pq-code-package/mlkem-c-aarch64/actions/runs/9543519253/job/26300276213?pr=60#step:5:66): warning: ignoring ‘#pragma CPROVER check’ [-Wunknown-pragmas]
   65 | #pragma CPROVER check disable "unsigned-overflow"
      | 
mlkem/poly.c:[67](https://github.com/pq-code-package/mlkem-c-aarch64/actions/runs/9543519253/job/26300276213?pr=60#step:5:68): warning: ignoring ‘#pragma CPROVER check’ [-Wunknown-pragmas]
   67 | #pragma CPROVER check pop
```

Reply:
Ah, thanks for the catch! It's because the extra quoting that I forgot to remove while removing the makefile macros.

Reply:
Thank you @potsrevennil, I can see the command now:
```
INFO  > make CROSS_PREFIX=aarch64-none-linux-gnu- test/bin/gen_KAT512
  CC      test/bin/gen_KAT512
mlkem/poly.c: In function ‘pqcrystals_kyber512_ref_scalar_compress_q_16’:
mlkem/poly.c:27: warning: ignoring ‘#pragma CPROVER check’ [-Wunknown-pragmas]
   27 | #pragma CPROVER check push
      | 
mlkem/poly.c:28: warning: ignoring ‘#pragma CPROVER check’ [-Wunknown-pragmas]
   28 | #pragma CPROVER check disable "unsigned-overflow"
      | 
mlkem/poly.c:30: warning: ignoring ‘#pragma CPROVER check’ [-Wunknown-pragmas]
   30 | #pragma CPROVER check pop
      | 
mlkem/poly.c: In function ‘pqcrystals_kyber5[12](https://github.com/pq-code-package/mlkem-c-aarch64/actions/runs/9543644196/job/26300636989?pr=60#step:5:13)_ref_scalar_compress_q_32’:
mlkem/poly.c:64: warning: ignoring ‘#pragma CPROVER check’ [-Wunknown-pragmas]
   64 | #pragma CPROVER check push
      | 
mlkem/poly.c:65: warning: ignoring ‘#pragma CPROVER check’ [-Wunknown-pragmas]
   65 | #pragma CPROVER check disable "unsigned-overflow"
      | 
mlkem/poly.c:67: warning: ignoring ‘#pragma CPROVER check’ [-Wunknown-pragmas]
   67 | #pragma CPROVER check pop
      |
```

Can one even go further and show the exact commands that were run?

Reply:
The exact command that were run is printed at the first line starting with the `INFO > ` prompt. Is this what you want ?

Reply:
No, I mean the commands that `make` runs

Reply:
ok, I removed some silent quotes, now it will not suppress the commands that `make` run

Reply:
Thank you!